source/security/enterprise-auth.txt

```diff
@@ -30,6 +30,7 @@ Enterprise Edition:
 - :ref:`Kerberos (GSSAPI) <gssapi-auth-mechanism>`
 - :ref:`LDAP (PLAIN) <plain-auth-mechanism>`
 - :ref:`MONGODB-OIDC <mongodb-oidc>`
+- :ref:`Kubernetes <java-sync-kubernetes>`
```

Comment:
Kubernetes should be added under the existing OIDC content, not as a separate auth mechanism. See [here](https://www.mongodb.com/docs/drivers/csharp/current/fundamentals/authentication/oidc/#kubernetes) for an example